apps/communications/dialer/test/marionette/lib/dialer.js

```diff
@@ -34,6 +37,7 @@ Dialer.Selectors = {
   two: '.keypad-key[data-value="2"]',
   three: '.keypad-key[data-value="3"]',
   keypadCallBarAddContact: '#keypad-callbar-add-contact',
+  keypadCallBarCall: '#keypad-callbar-call-action',
```

Comment:
Yeah I though that maybe they can be used in the future, but for this test, I can remove them